sycl/test/esimd/on-device/matrix_transpose_glb.cpp

```diff
@@ -10,6 +10,7 @@
 // REQUIRES: gpu
 // RUN: %clangxx-esimd -fsycl %s -o %t.out
 // RUN: %ESIMD_RUN_PLACEHOLDER %t.out
+// XFAIL: gpu && (level_zero || opencl) && linux
```

Comment:
```suggestion
// XFAIL: linux
```
Why this test is expected to fail?
If it's not expected, please, open an issue to address this failure later.

Reply:
yes, will open an issue to track